### DISCUSSION TITLE
File Upload Block Picker - Block Preview

### DISCUSSION THREAD

**Discussion by Unrivaled9194** [standardnotes/forum#3559] [Suggestions]
Love that we can now upload a file inline in a note via the block picker! But the execution seems a bit clunky as there's no way to change how it appears within the note. 

For example, if you upload a PDF it will display a large preview of the PDF but there is no way to minimize the preview to either just the title of the file or a smaller card view of some sort.  

![image](https://github.com/standardnotes/forum/assets/41435929/321051d8-4f6c-4fed-afd7-7d54c3e404bb)

Same with a file that is in a format that the note doesn't recognize (such as an excel file). It will simply displays a large block like below that says the file can't be previewed. Again it would be handy to be able to toggle that preview off and display a simple file name. 

![image](https://github.com/standardnotes/forum/assets/41435929/44554515-bded-4e65-a829-7209fced821f)

Keep up the great work!

**Comment by tkassing** [comment]
This would be my preference as well.  I just want to see the file name or an icon in the note. The "file can't be previewed" takes a lot of space as opposed to a simple icon, so does the PDF preview.
